components/logins/src/login.rs

```diff
@@ -32,15 +32,16 @@
 //!     implementations follow this flow.  Still, we want application-services to support it.
 //!   - If the user chooses to autofill, we decrypt the logins into a `Vec<Login>`.  We need to
 //!     decrypt at this point to display the username and autofill the password if they select one.
+//!   - When the user selects a login, we can use the already decrypted data from `Login` to fill
```

Comment:
nit: This is a note about this logins autofill explanation in general and not your addition to it, it might be worth noting that this is an explanation of the flow in a-c since we don't have ultimate control over how the api will be used. And even at present this doesn't apply to iOS as they don't even use `get_by_base_domain()`.